isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -755,9 +793,11 @@ namespace Isis {
 
     // notify CubeSerialNumbers of the loss of this point
     foreach(ControlMeasure * measure, point->getMeasures()) {
-      measureDeleted(measure);
+      emit measureRemoved(measure);
```

Comment:
This needs to be changed back to measureDeleted. This doesn't update the network's graph structure, and measureDeleted will emit the measureRemoved signal